### PR TITLE
fix: add index

### PR DESCRIPTION
**Pull request by marcod1419**
I want to add code to your repo!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<html>
+
+<body>
+  <h1>Hello, world!</h1>
```
Comment (IsabellaValentina) [inline index.html]: I hate that

**Review by IsabellaValentina — CHANGES_REQUESTED** [review]
nah

File: index.html (hunk 1 above)
Comment (marcod1419) [reply index.html]: Ok what do you suggest

Comment (IsabellaValentina) [reply index.html]:
```suggestion
  <h1>Hello, people!</h1>
```

This!

**Review by IsabellaValentina — APPROVED** [review]

**Pull request merged**